src/component/visualMap/ContinuousView.ts

```diff
@@ -109,6 +109,9 @@ class ContinuousView extends VisualMapView {
 
     private _api: ExtensionAPI;
 
+    private _ownHoverLinkFromSeriesMouseOver = (e: ElementEvent) => this._hoverLinkFromSeriesMouseOver(e);
+
+    private _ownHideIndicator = () => this._hideIndicator();
```

Comment:
I think we can refer to the [`SliderZoomView`](https://github.com/apache/echarts/blob/master/src/component/dataZoom/SliderZoomView.ts#L135-L141) and use the same strategy.

Note that since some codes use `this.api` rather than `this._api`, you may need to change them to `this._api` after implementing the `init` function.